docs/types.md

```diff
@@ -767,7 +767,7 @@ Submission limits are not enforced for course graders.
 ## File Specification (FileSpec)
 
 A file specification (FileSpec) defines how to access a specific file (or dir).
-At it easiest it is just the path to a file, but can also specify things like a URL or Git repository.
+At its easiest, it is just the path to a file, but can also specify things like a URL, a Git repository, or a glob pattern for matching multiple files or directories.
```

Comment:
"glob pattern" should link to some documentation on globs (Wikipedia, probably).